docs/architecture/ingest.md

```diff
@@ -0,0 +1,88 @@
+# Ingest
+
+This is a component that is responsible for ingesting and processing data.
+
+It exposes a GRPC API for Osmosis node clients to send data to be ingested.
+
+See [this ADR](https://www.notion.so/osmosiszone/ADR-006-Streaming-GRPC-SQS-Ingest-4e3b2ff7d23e43e2a1f3c43adc3c26bc) for details about the design.
+
+The node pushes only the pool data that is changed within a block.
```

Comment:
Suggestion: adding a link pointing to https://github.com/osmosis-labs/osmosis/blob/970db14d2ee14b4301bc6ebf6d620fa907923105/ingest/sqs/service/grpc_client.go#L42 for reference purposes.

Reply:
[df05da0](https://github.com/osmosis-labs/sqs/pull/281/commits/df05da0d66fbb33884d7ca755e1249cd5a334184)